common/persistence/cassandra/errors.go

```diff
@@ -199,11 +201,27 @@ func extractCurrentWorkflowConflictError(
 
 	actualCurrentRunID := gocql.UUIDToString(record["current_run_id"])
 	if actualCurrentRunID != requestCurrentRunID {
+		binary, _ := record["execution_state"].([]byte)
+		encoding, _ := record["execution_state_encoding"].(string)
+		executionState := &persistencespb.WorkflowExecutionState{}
+		if state, err := serialization.WorkflowExecutionStateFromBlob(binary, encoding); err != nil {
+			// noop, this means execution state cannot be parsed
+		} else {
+			executionState = state
+		}
```

Comment:
isn't it more clear to do just:
if ...; err == nil {
  executionState = state
}

Reply:
sure i can to the above